pkg/controllermanager/apis/config/v1alpha1/defaults.go

```diff
@@ -140,10 +140,7 @@ func SetDefaults_ClientConnectionConfiguration(obj *componentbaseconfigv1alpha1.
 // SetDefaults_LeaderElectionConfiguration sets defaults for the leader election of the Gardener controller manager.
 func SetDefaults_LeaderElectionConfiguration(obj *LeaderElectionConfiguration) {
 	if obj.ResourceLock == "" {
-		// TODO: change default to leases after a few releases
-		// make sure, we had configmapsleases as default for a few releases before migrating to leases to ensure,
-		// all users had at least one version running with the hybrid lock to avoid split-brain scenarios when migrating.
-		obj.ResourceLock = resourcelock.ConfigMapsLeasesResourceLock
```

Comment:
In the release note: Can you mention the minimum Gardener version that is required before it's possible to update to `v1.19`?

Reply:
Done 👍